src/components/BlockInsert.js

```diff
@@ -44,12 +44,12 @@ class BlockInsert extends React.Component<Props, State> {
   };
 
   componentDidMount = () => {
-    window.addEventListener("mousemove", this.handleMouseMove);
+    typeof window != 'undefined' && window.addEventListener("mousemove", this.handleMouseMove);
```

Comment:
Please update to `!==` in this file, if you push again hopefully I've changed the settings so that CI will build the PR now